session-loader/Development/IDE/Session.hs

```diff
@@ -312,7 +313,7 @@ loadSession dir = do
             -- update xports map
             extras <- getShakeExtras
             let !exportsMap' = createExportsMap $ mapMaybe (fmap hirModIface) modIfaces
-            liftIO $ modifyVar_ (exportsMap extras) $ return . (exportsMap' <>)
+            liftIO $ modifyVar_ (exportsMap extras) $ evaluate . (exportsMap' <>)
```

Comment:
Definitely! We should probably add an HLint rule for this - or wrap modifyVar_ into modifyVar_' and only use those.

Reply:
An hlint rule would be great